vllm/worker/vineyard_llm_cache.py

```diff
@@ -192,7 +195,7 @@ def from_envs(
         head_size = model_config.get_head_size()
         num_kv_heads = model_config.get_num_kv_heads(parallel_config)
         num_layers = model_config.get_num_layers(parallel_config)
-        token_nbytes = num_layers * num_kv_heads * head_size * 2
+        token_nbytes = num_layers * num_kv_heads * head_size * 4  # sizeof(float16) * 2 (kv)
```

Comment:
nit: better to use sizeof(float16) * 2 directly in the code.

this should be a bug from previous change. Any problems we've seen?

Reply:
Seems only request volume and cache calculation are affected. nice catch!